pkg/cloud/azureprovider.go

```diff
@@ -73,6 +73,77 @@ var (
 	mtStandardN, _ = regexp.Compile(`^Standard_N[C|D|V]\d+r?[_v\d]*[_Promo]*$`)
 )
 
+var azureRegions = []string{
```

Comment:
Would you include a comment explaining how to generate this list again in the event of Azure's regions changing?

Reply:
Done! And glad you asked -- I had used a less-authoritative source initially and it was missing regions. New list is taken from the Azure CLI